### DISCUSSION TITLE
Checksum for api.community.zip

### DISCUSSION THREAD

**Discussion by cpach** [envkey/envkey#82] [Ideas and feedback]
Looks like the published checksum for version 2.3.2 of api.community.zip is incorrect…?

The checksum contained in the file https://envkey-releases.s3.amazonaws.com/apicommunity/release_artifacts/2.3.2/d49329bf2ab7da3f.version.shasum is d49329bf2ab7da3f00574a6ab9e3ca61cf0c2766e5aac281eb50fed4986ce28c

When I run `shasum -a 256 api.community.zip` I get this result:

```
18cd1747071764aae2181a8d3d17fffa64de2b8020786cefd4545cda3b44487c  api.community.zip
```

**Comment by danenania** [comment]
Hi @cpach, the shasum is derived from the compiled code (which is wrapped in a binary) and the Dockerfile rather than the final zip artifact. It's intended for use by our build process rather than an integrity check for end users. You can instead verify the release signature with `minisign` for this purpose. Details here: https://docs-v2.envkey.com/docs/verifying-releases

**Comment by cpach** [comment]
Ok, got it! Thank you for clariyfing.

My use-case was to validate the integrity of the .zip in an Ansible playbook.

You might want to include a checksum of the .zip file too. (AFAICT Ansible doesn’t currently support minisign signatures.)

**Comment by danenania** [comment]
I see, I can understand how it's a bit misleading. We'll look at generating checksums for the zip files as well--thanks for flagging this. I'm going to move this to discussions so we don't lose track.
